### ISSUE TITLE
Adjust "Our Sponsors" list to display as shown

### ISSUE THREAD

**Issue by katrinasperry** [codeforpdx/website#41]
Create PR that merges to sponsor_list-redesign branch

![image](https://user-images.githubusercontent.com/29134018/142134070-5d19d527-fa19-43d5-85f9-7adbf96d4132.png)

NOTE: all sponsor logos are already on the current site - keep them as is!

The sponsors html can be found in: `sections/sponsors.html`
The mardown files that contain all the sponsor info can be found in the md files located in `_sponsors` <- Probably don't need to modify those files, but good to know where it is.

**Comment by curtisbarnard** [comment]
I can probably take this one on. A few questions as this is the first time I've worked with Jekyll and on a GitHub branch:

- In the sponsor_list-redesign branch there is no sections folder with the sponsors.html file. Should I create that file in that branch? Or should I modify the appropriate section in the home.html of that branch?
- It seems I can modify with the current classes in main.scss or I could create a new class. Is there a preference?
- Do I need to worry about the header font or is that covered in another issue already?
- To have the logos sized as the photo above and keep consistent css styling the logo files will need to be cropped, should I do that? Or should I have a class for each logo to tweak it's sizing to match above.
- Attached is what I did already by modifying the grid-container class.

![sponsors](https://user-images.githubusercontent.com/100104319/159944451-79df9024-d0e9-4f1e-b397-768cd5f4a9af.JPG)
